src/systems/systemItemsTable.component.tsx

```diff
@@ -116,39 +177,113 @@ export function SystemItemsTable(props: SystemItemsTableProps) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isLoading, itemsData]);
```

Comment:
I think this might also need updating (I would remove the eslint disable above, fix it then add it back in having removed the catalogueItemList again).

Reply:
There are missing dependencies but vscode and the linter don't complain because its disabled on this line with `// eslint-disable-next-line react-hooks/exhaustive-deps` which was only added to prevent re-renders for catalogueItemList (see the comment above it), but you have added more dependencies that should be added.